meta-balena-honister/README.md

```diff
@@ -0,0 +1,11 @@
+# Balena.io layer for Poky Honister supported boards
+
+## Description
+This repository enables building balena.io for sumo supported machines.
```

Comment:
s/sumo/honister

Reply:
Thank you for spotting this Alex, it's now fixed in https://github.com/balena-os/meta-balena/pull/2368/commits/b9625511a52a5e4dc0a76496649791299b4fe130